rising/transforms/grid.py

```diff
@@ -0,0 +1,94 @@
+from typing import Sequence, Union, Dict, Tuple
+
+import torch
+
+from abc import abstractmethod
+from torch import Tensor
+
+from rising.transforms import AbstractTransform
+from rising.utils.affine import get_batched_eye, matrix_to_homogeneous
+
+
+__all__ = ["GridTransform", "StackedGridTransform"]
+
+
+class GridTransform(AbstractTransform):
```

Comment:
What if I somehow get my grid from somewhere else and just want to use it? can we have an optional argument in here?

Reply:
same as above?

Reply:
I don't get it

Reply:
Sorry, thought the upper comment was at the same position. ^^
Yeah, we can add that. I think I would prefer an extra Transform for that, just like the Affine Transform has